webportal/src/app/cluster-view/services/services.component.js

```diff
@@ -22,4 +22,51 @@ const servicesComponent = require('./services.component.ejs');
 $("#sidebar-menu--cluster-view").addClass("active");
 $("#sidebar-menu--cluster-view--services").addClass("active");
```

Comment:
move these before `$(document).ready`, and use single quotes in js please

Reply:
Solved